### ISSUE TITLE
Choose the arrow style independently of the scheme

### ISSUE THREAD

**Issue by Rangi42** [fltk/fltk#1117]
**Is your feature request related to a problem? Please describe.**

In FLTK 1.3, widgets like `Fl_Spinner` drew their arrows via `@` symbols, using functions in `fl_symbols.cxx`. Now in FLTK 1.4, they call functions in `fl_draw_arrow.cxx`, which chooses an arrow style (simple triangle vs indented chevron) based on the current scheme.

I use the `"gtk+"` scheme in some custom themes (some of which were ported to [fltk-rs/fltk-theme](https://github.com/fltk-rs/fltk-theme)), because of the other visual details it affects (the dropdown arrow style in `Fl_Choice`, the checkbox/radio button appearance, the gripper on scrollbars, etc). But now that it makes arrows look like chevrons, the themes look less like the platforms they're meant to imitate.

**I am using FLTK**
 
1.4.0rc3

**Describe the solution you'd like**

I would appreciate a global setting for the arrow style, e.g. `Fl::arrow_style` (going along with similar appearance options like `Fl::scrollbar_width`). Perhaps `Fl::scheme` would call this to set it initially, but then the user could call `Fl::arrow_style` afterwards to reset it.

**Describe alternatives you've considered**

The alternative of leaving arrows as they are would require me to either live with the unusual-for-their-platform arrows, or reimplement FLTK's provided widgets just for the sake of their arrow appearance.

**Additional context**

FLTK 1.4's chevron-style arrows:

![image](https://github.com/user-attachments/assets/dbac80d7-ac8f-4b65-b7b5-5186ebdf7766)

**Comment by Albrecht-S** [comment]
Thanks for the report, I'm looking into it...

**Comment by Albrecht-S** [comment]
TL;DR: After investigation I decided to revert the "chevron style" drawing for scheme "gtk+" and to use the "triangle" arrow shape for "gtk+" like for almost all other schemes (except maybe "oxy").

**_History:_** we had different "hand made" arrow drawing code - supposedly copied and modified (or not) - at several places in widget implementations. Some of them used polygon drawing or similar, others used FLTK "symbols" as the OP @Rangi42 reported. This turned out to be a maintenance nightmare and made adding different schemes (like "oxy" in FLTK 1.4.0) almost impossible.

**_The solution_** was to consolidate and unify "arrow drawing" and drawing of other UI elements like check marks and radio buttons across widgets and to make these drawing methods scheme specific. This was a first step in the direction of making scheme implementation easier which will be pursued further in 1.5.0 or later.

**_Regarding this issue:_** I believe that the reason for this issue has been fixed with commit dc2c53333cf598d3b7fd5d183da671671dff2896 and we can close this issue. An option to select different arrow types doesn't seem appropriate any more.

@Rangi42 Please close this issue if you agree or tell us if you think we need to do more...

**Comment by Rangi42** [comment]
> After comparison with older versions I realized that the gtk+ specific "chevron style" was previously used *exclusively* in Fl_Scrollbar. Unfortunately I had picked this style as template for all arrows.

This makes sense, and just using triangles sounds like a good solution to me! Thank you very much. :)
